mcs/build/profiles/winaot.make

```diff
@@ -34,7 +34,8 @@ FRAMEWORK_VERSION = 2.1
 # the tuner takes care of the install
 NO_INSTALL = yes
 AOT_FRIENDLY_PROFILE = yes
-ALWAYS_AOT = yes
+ALWAYS_AOT_BCL = yes
+ALWAYS_AOT_TESTS = yes
 MOBILE_PROFILE = yes
 NO_VTS_TEST = yes
 NO_SRE=1
```

Comment:
If I understand the intention correctly, you forgot you update `testing_aot_full.make` to have ALWAYS_AOT_BCL = yes

Reply:
Why is there no `ALWAYS_AOT_TESTS = yes` in `testing_aot_full.make` config file?

Reply:
As far as my changes show, ALWAYS_AOT_TESTS was always defined in testing_aot_full.make. Maybe github playing some tricks ?